### DISCUSSION TITLE
No Access on local network but access from outside

### DISCUSSION THREAD

**Discussion by harrischrisa** [rustdesk/rustdesk#7281] [Q&A] [answered]
I am using the latest Android clients to access my computers using default Rustdesk severs. From outside my home network I can connect, but if on my home network (same as computers I am trying to connect to) I get "failed to connect to relay server" 
![img1](https://github.com/rustdesk/rustdesk/assets/16251436/4923b4e7-5197-4af5-8044-7aa9be762467)

If I turn off WiFi and use the telephone network I can connect.

 I have a WiFi tablet which will not connect on same network. If I disconnect local WiFi and connect to my phone hotspot (which is feeding from my home network) I can connect.

If I use the Rustdesk client on one of the Windows PC which I was trying to connect to, to connect to the second Windows PC on the same network I can connect but takes 3 times longer than connecting to an external computer at my son's house.

I have protected access to my network by means of a Caddy reverse proxy.

What do I have to do to get full access to my computers?

Thanks

**Comment by harrischrisa** [comment]
Can anyone assist?

**Comment by xlionjuan** [comment]
I wrote this
https://rustdesk.com/docs/en/self-host/nat-loopback-issues/

**Comment by harrischrisa** [comment, marked as answer]
Thanks for the discussion paper. It was very helpful.

In the end I realised I had install an Android client from Google Play. It was old. I installed the latest from Github and everything works.
